docker-compose/envs/common-blockscout.env

```
@@ -1,19 +1,19 @@
 ETHEREUM_JSONRPC_VARIANT=geth
-ETHEREUM_JSONRPC_HTTP_URL=https://testnet.rpc.hemi.network/rpc
+ETHEREUM_JSONRPC_HTTP_URL=int02.testnet.rpc.hemi.network/rpc
```

The protocol must be specified in the three env vars.

```suggestion
ETHEREUM_JSONRPC_HTTP_URL=https://int02.testnet.rpc.hemi.network/rpc
```

---

Changes applied